K2Field.K2NE.ServiceBroker/ServiceObjects/ADOQuerySO.cs

```diff
@@ -0,0 +1,111 @@
+﻿using K2Field.K2NE.ServiceBroker.Helpers;
+using SourceCode.SmartObjects.Services.ServiceSDK.Objects;
+using SourceCode.SmartObjects.Services.ServiceSDK.Types;
+using CLIENT = SourceCode.Workflow.Client;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using SourceCode.Workflow.Management;
+using System.Data.SqlClient;
+
+using SourceCode.Data.SmartObjectsClient;
+using System.Text.RegularExpressions;
+
+
+namespace K2Field.K2NE.ServiceBroker.ServiceObjects
+{
+    class ADOQuerySO : ServiceObjectBase
```

Comment:
Please rename class to ADOSMOQuery to make clear that it is for SMO ADO queries.